voxblox/src/utils/protobuf_utils.cc

```diff
@@ -31,7 +31,7 @@ bool writeProtoMsgCountToStream(uint32_t message_count,
   CHECK_NOTNULL(stream_out);
   CHECK(stream_out->is_open());
   stream_out->clear();
-  stream_out->seekg(0, std::ios::beg);
+  // stream_out->seekg(0, std::ios::beg);
```

Comment:
why is this commented out, if it is unneeded shouldn't it just be removed

Reply:
Good catch!